### ISSUE TITLE
No wait cursor when launching application

### ISSUE THREAD

**Issue by hzulla** [flathub/org.signal.Signal#341]
See https://github.com/flatpak/flatpak/issues/5050, but this appears to be an app-specific problem.

When you launch an application on Ubuntu's default variant of Gnome, a wait cursor will tell the user that the application is loading. The wait cursor disappears once the application windows appears.

This works for most flatpaks I have installed, but not for Signal Desktop 5.56.0.

With Signal Desktop, the cursor does not change. The Signal Desktop application will launch eventually, but until then, there is no visual feedback to the user that the computer is busy launching the application.

**Comment by bermeitinger-b** [comment]
This might be fixed by adding `StartupNotify=true` to the .desktop file.

I don't know if it's there upstream and if this is a flathub specific issue. It could be, but we're using the same file as upstream.

**Comment by hzulla** [comment]
Still happening with 5.61.1

**Comment by bermeitinger-b** [comment]
> This might be fixed by adding StartupNotify=true to the .desktop file.

Did you try this?

**Comment by hzulla** [comment]
No, but just tried it. Indeed, a wait cursor then appears, but it remains on screen long after the application window has fully appeared. Eventually, the wait cursor disappears, but its behaviour does not align with what the user sees on the screen.

**Comment by hzulla** [comment]
The normal behaviour should be:

- User clicks application icon in launcher
- wait cursor appears 
- application window appears and wait cursor disappears at the same time.

**Comment by hzulla** [comment]
Still happening with 5.62.0

**Comment by bermeitinger-b** [comment]
Would be weird if not because not change regarding this was done.
